.github/workflows/atqa-framework-on-checks-branch.yml

```diff
@@ -0,0 +1,202 @@
+        id: prevTag
+
+      - name: Get sources for current tag
+        uses: actions/checkout@v3
+        with:
+          ref: '${{ steps.prevTag.outputs.VALUE }}'
+
+      - name: Read .nvmrc for keeper-wallet
+        run: echo ::set-output name=NODE_VERSION::$(cat .nvmrc)
+        id: nvm_keeper
+
+      - name: Setup NodeJS
+        uses: actions/setup-node@v2
+        with:
+          node-version: '${{ steps.nvm_keeper.outputs.NODE_VERSION }}'
+
+      - name: Install dependencies
+        run: yarn --frozen-lockfile
+
+      - name: Create empty config.json
+        run: echo '{}' > config.json
+
+      #Build master with tags
+
+      - name: Build
+        run: yarn run dist
+
+      #Copy master with tag bins to buffer folder
+
+      - name: Prepare extension bins for uploading
+        run: |
+          mkdir ./extension
+          cp -Rv ./dist/opera ./dist/chrome ./dist/edge ./extension/
+
+      - name: Save extension artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: extension-init-bins
+          #          path: ./atqa-framework/extension/
+          path: ./extension/
+
+        #Set up current branch
+
+      - name: Get Current branch source
+        uses: actions/checkout@v3
+
+      - name: Read .nvmrc for keeper-wallet
+        run: echo ::set-output name=NODE_VERSION::$(cat .nvmrc)
+        id: nvm_keeper_v2
+
+      - name: Setup NodeJS for current branch
+        uses: actions/setup-node@v2
+        with:
+          node-version: '${{ steps.nvm_keeper_v2.outputs.NODE_VERSION }}'
+
+      - name: Install dependencies
+        run: yarn --frozen-lockfile
+
+      - name: Create empty config.json
+        run: echo '{}' > config.json
+
+      #Build extension for current branch
+
+      - name: Build
+        run: yarn run dist
+
+      #Set up atqa-framework
+
+      - name: Prepare nvm for atqa-folder
+        working-directory: ./atqa-framework
+        run: echo ::set-output name=NODE_VERSION::$(cat .nvmrc)
+        id: nvm_atqa_framework
+
+      - name: Setup NodeJS for atqa-framework
+        uses: actions/setup-node@v2
+        with:
+          node-version: '${{ steps.nvm_atqa_framework.outputs.NODE_VERSION }}'
+
+      - name: Install dependencies
+        working-directory: ./atqa-framework
+        run: yarn --frozen-lockfile
+
+        #Setting up Selenoid
+
+      - name: Setting up Selenoid ecosystem
+        working-directory: ./atqa-framework
+        run: docker compose up -d
+
+      - name: Pulling browser images
+        working-directory: ./atqa-framework
+        run: |
+          docker pull selenoid/vnc_opera:88.0
+          docker pull selenoid/vnc_chrome:102.0
+          docker pull browsers/edge:103.0
+
+      #init DotEnv and start tests
+
+      - name: Init buffer and extension init folders
+        working-directory: ./atqa-framework
+        run: |
+          sudo mkdir -m 777 buffer/
+          sudo mkdir -m 777 extension/
```

Comment:
I doubt you ever need to use sudo in this file. And why do you need 777 permissions?

Reply:
I know the root cause but I suggest don't touch FROM magic of the native images builded by aerokube team

if we want build our own images with specific user let's build it from sources, but for me it's overhead, why 777 is bad?